infrastructure/src/main/kotlin/DetektPlugin.kt

```diff
@@ -71,27 +61,32 @@ private fun Project.configureDetektAllTasks(extension: RedmadrobotExtension, inf
         description = "Runs over whole code base without the starting overhead for each module."
     }
 
-    detektCreateBaselineTask(extension, infrastructureRootProject, "detekt${BASELINE_KEYWORD}All") {
+    detektCreateBaselineTask(
+        extension,
+        infrastructureRootProject,
+        "detekt${DetektPlugin.BASELINE_KEYWORD}All",
+    ) {
         description = "Runs over whole code base without the starting overhead for each module."
     }
 
     if (project.isInfrastructureRootProject) {
-        val variantRegex = Regex("detekt($BASELINE_KEYWORD)?([A-Za-z]+)All$")
-        val taskRegex = Regex("^(${project.path.orEmpty()}:)?$variantRegex")
+        val variantRegex = Regex("detekt(${DetektPlugin.BASELINE_KEYWORD})?([A-Za-z]+)All$")
+        val taskRegex = Regex("^(:?${project.name.orEmpty()}:)?$variantRegex")
```

Comment:
Will project.name include parent projects also? For example, we have a hierarchy:
```
libs <-- parent project
libs.lib1 <-- project where detekt plugin applied
```
Will `project.name` return `libs:lib1` or `lib1`?

Reply:
Good point! No, that will be fixed.  Hadn't thought of that.